requirements-devel.txt

```diff
@@ -11,7 +11,7 @@ click==8.1.7
 codespell==2.2.6
 colorama==0.4.6
 coverage==7.4.0
-craft-application==2.1.0
+craft-application @ git+https://github.com/canonical/craft-application.git@CRAFT-2601-verbosity-level
```

Comment:
Are you planning to wait for a new craft-application release before landing this?

Reply:
Yes